pkg/steps/clusterinstall/template.go

```diff
@@ -334,6 +334,10 @@ objects:
           setup-google-cloud-sdk
         fi
 
+        echo "Running must-gather..."
+        mkdir -p /tmp/artifacts/pre-test-must-gather
+        oc --insecure-skip-tls-verify adm must-gather --dest-dir /tmp/artifacts/pre-test-must-gather >/tmp/artifacts/pre-test-must-gather/must-gather.log
```

Comment:
not sure we actually need `insecure-skip-tls-verify`.  I'd expect the kubeconfig to include the custom cert.  But whatever, seems orthogonal to this PR, so I'm just rolling forward with the same `must-gather` call we use during `teardown`.